src/Telegram.Bot/ITelegramBotClient.cs

```diff
@@ -474,10 +474,11 @@ public interface ITelegramBotClient
         /// </summary>
         /// <param name="chatId"><see cref="ChatId"/> for the target group</param>
         /// <param name="userId">Unique identifier of the target user</param>
+        /// <param name="untilDate"><see cref="DateTime"/> when the user will be unbanned. If user is banned for more than 366 days or less than 30 seconds from the current time they are considered to be banned forever</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
         /// <returns><c>true</c> on success.</returns>
         /// <see href="https://core.telegram.org/bots/api#kickchatmember"/>
-        Task<bool> KickChatMemberAsync(ChatId chatId, int userId,
+        Task<bool> KickChatMemberAsync(ChatId chatId, int userId, DateTime untilDate = default(DateTime),
```

Comment:
@Olfi01 If I always need to pass it values such as `DateTime.UtcNow`, it makes more sense to me to change parameter name to `utcUntilDate`. Instructs the clients to pass it as UTC time.